README.md

```diff
@@ -16,7 +16,7 @@ Oreboot+QEMU for RISC-V HiFive Unleased:
 
 [![asciicast](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2.svg)](https://asciinema.org/a/XnWkMWTABuajsbGPMMTefjuZ2)
 
-Oreboot+QEMU for AST2500:
+Oreboot+QEMU for ARM:
```

Comment:
I think you need a note in here somewhere about export OREBOOT into the environment. That made x86 builds work anyway.